src/routes/_layout.svelte

```diff
@@ -1,9 +1,53 @@
+<script>
+    import AppBar from "../components/appBar/AppBar.svelte";
+    import { t } from "../translations/translation";
+    let y;
+</script>
+
+<svelte:window bind:scrollY={y} />
+
+<nav>
+    <AppBar />
+</nav>
+
 <main>
+    <div class="gradient" />
+    <img src="images/boritokep.jpg" alt="borítókép" />
```

Comment:
nekem az is jó ha az altok nem translationből jönnek, de akkor legyen angol nyelven :)

Reply:
Illetve talán fájlok nevei is jobb ha angolok (igazából nekem a magyar is jó csak legyen konzistens)